src/servers/Cargo.toml

```diff
@@ -80,6 +79,7 @@ notify.workspace = true
 object-pool = "0.5"
 once_cell.workspace = true
 openmetrics-parser = "0.4"
+rust-embed = { version = "8.2", optional = true, features = ["debug-embed"] }
```

Comment:
Why do we bump the version and move it to this place?

I also notice that the dependencies in the file are not sorted in lexical order. You can also fix this.

Reply:
Thank you!